docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -886,7 +886,7 @@ The table below show the OAuth provider and their setting with the default value
 
 To prevent synchronization of organization roles for a specific OAuth integration, you can set the `skip_org_role_sync` option to `true`. Please note that there is also a separate setting called `oauth_skip_org_role_update_sync` which has a different scope. While `skip_org_role_sync` only applies to the specific OAuth provider, `oauth_skip_org_role_update_sync` is a generic setting that affects all configured OAuth providers.
 
-The setting `oauth_skip_org_role_update_sync` will be deprecated in favor of provider-specific settings.
+> **Note:** There is a separate setting called `oauth_skip_org_role_update_sync` which has a different scope. While `skip_org_role_sync` only applies to the specific OAuth provider, `oauth_skip_org_role_update_sync` is a generic setting that affects all configured OAuth providers.
```

Comment:
Doesn't it duplicates the paragraph directly above?
```suggestion
```

Reply:
working on a wording of this